### PR TITLE
[DYN-4997] Fix multiple Trust Warning related bugs

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

[DYN-4997](https://jira.autodesk.com/browse/DYN-4997)

- When Disable trust warning toggle set to true, file trust warning should not show up : **Fixed**
- Tooltip arrow direction (Similar in [DYN-4881](https://jira.autodesk.com/browse/DYN-4881)) : **Fixed**
- Wrapping tooltip from hovering over question mark " ->" : **Fixed**
- Update tooltip to: `The file's current location will be added to trusted locations. You can manage this in Preferences -> Security.` : **Fixed**
- Trust path should take effect recursively : **Fixed**
- Pop-up at the center of the Automatic run-mode bar makes sense : **Fixed**
- Fix the bug that the edit path or add path dialog will not close at the first trial and re-popup. : **Fixed**

Snapshots:

Tooltips, Popup Centered:

tooltip

Recursively Trust Locations, Add Path dialog bug fix:

recursive

Disable Trust Warning Toggle:

toggle

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

- Fix multiple Trust Warning related bugs

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

**Comment by QilongTang** [comment]
I think my PR introduced merge conflicts

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 1]
```diff
@@ -1607,7 +1607,7 @@ private void Open(object parameters)
                 var directoryName = Path.GetDirectoryName(filePath);
                 //Checks if the file that is being opened is in the trusted list.
                 bool bShowFileTrustWarning = false;
-                if(!PreferenceSettings.TrustedLocations.Contains(directoryName) && DynamoModel.IsTestMode == false)
+                if(!PreferenceSettings.IsTrustedLocation(directoryName) && !DynamoModel.IsTestMode && !PreferenceSettings.DisableTrustWarnings)
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs:1610]: 👍🏻

**Comment by QilongTang** [comment]
Cool, let us know when this PR is ready for review

File: src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs [hunk 2]
```diff
@@ -144,22 +145,22 @@ private void SetupFooterNotificationItems()
             footerItems[0] = new FooterNotificationItem()
             {
                 NotificationCount = 0,
-                NotificationImage = "/DynamoCoreWpf;component/UI/Images/error.png",
-                NotificationToolTip = "Click to cycle through nodes with errors.",
+                NotificationImage = Resources.FooterNotificationErrorImage,
+                NotificationToolTip = Resources.FooterNotificationErrorTooltip,
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs:149]: These look unrelated to the bug, but good changes to be included. Did you find these while making the bug fixes?

File: src/DynamoCoreWpf/ViewModels/FileTrust/FileTrustWarningViewModel.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@ namespace Dynamo.Wpf.ViewModels.FileTrust
     public class FileTrustWarningViewModel : ViewModelBase
     {
         private bool showWarningPopup;
-        private double PointerHorizontalOffset = 15;
+        private double PointerHorizontalOffset = 60;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/FileTrust/FileTrustWarningViewModel.cs:13]: could you add a comment saying, the number here will make the pointer stand in the middle?

**Review by QilongTang — APPROVED** [review]
LGTM with a few comments

File: src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs (hunk 2 above)
Comment (zeusongit) [reply src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs:149]: Yes I found it while fixing the tooltips direction

**Comment by QilongTang** [comment]
https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/6313/ passed but not reflected on this PR check. Merging.

**Pull request merged**